src/Classes/Item.lua

```diff
@@ -242,15 +243,16 @@ function ItemClass:ParseRaw(raw)
 				elseif specName == "Limited to" and self.type == "Jewel" then
 					self.limit = tonumber(specVal)
 				elseif specName == "Variant" then
-					if not self.variantList then
-						self.variantList = { }
+					if not self.variantNameList then
+						self.variantNameList = { }
+						self.variantList = self.variantList or { Base = { } }
```

Comment:
```suggestion
						self.variantList = self.variantList or { base = { } }
```
Fields seem to typical start with lowercase in this project needs changing elsewhere.

Reply:
This is to match the other default names of "Two" "Three" "Four" etc which start with a capital, but I am indifferent and can change it